internal/uvm/create_lcow.go

```diff
@@ -466,10 +466,16 @@ func makeLCOWVMGSDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_
 	if uvm.scsiControllerCount > 0 {
 		logrus.Debug("makeLCOWVMGSDoc configuring scsi devices")
 		doc.VirtualMachine.Devices.Scsi = map[string]hcsschema.Scsi{}
+		for i := 0; i < int(uvm.scsiControllerCount); i++ {
+			doc.VirtualMachine.Devices.Scsi[guestrequest.ScsiControllerGuids[i]] = hcsschema.Scsi{
```

Comment:
we are guaranteed that `i < len(guestrequest.ScsiControllerGuids)`, right?

Reply:
yeah, we have a check in `verifyOptions` that will return an error if the count is more that `MaxSCSIControllers`, which is set to `len(guestrequest.ScsiConrollerGuids)`